app/models/ingest_job.rb

```diff
@@ -717,6 +733,12 @@ def generate_success_email_array
     when :differential_expression
       message << "Differential expression calculations for #{params_object.cluster_name} have completed"
       message << "Selected annotation: #{params_object.annotation_name} (#{params_object.annotation_scope})"
+    when :render_expression_arrays
+      matrix_name = params_object.matrix_file_path.split('/').last
+      matrix = study.expression_matrices.find_by(name: matrix_name)
+      genes = Gene.where(study_id: study.id, study_file_id: matrix.id).count
+      message << "Image Pipeline data pre-rendering completed for '#{params_object.cluster_name}' completed"
```

Comment:
Two refinements:
* Standardizing on double quotes to demarcate a special phrase in text would help greppability -- no need to wonder which quote to search; double is more conventional
* Remove redundant "completed

```suggestion
      message << "Image Pipeline data pre-rendering completed for \"#{params_object.cluster_name}\""
```

Also, would it be trivial to tack on the runtime?  It seems contextual here.

Reply:
Total parse time is default on all jobs and is the first message printed (line 677):
```
message = ["Total parse time: #{get_total_runtime}"]
```

Reply:
[81763b0](https://github.com/broadinstitute/single_cell_portal_core/pull/1634/commits/81763b0d5ce57fbb1355cce3b9e39c97052af5c4)